### ISSUE TITLE
adding taxonomy info not working when run in nucleotide mode

### ISSUE THREAD

**Issue by AstrobioMike** [AstrobioMike/GToTree#91]
when trying to add NCBI taxonomy information while running in nucleotide mode (`-z`), it doesn't seem to be pulling the taxid info, as these messages are popping up and there is no tax added in the output alignment or tree files, need to track down why:

```
 ##############################################################################
 ####                   Adding more informative headers                    ####
 ##############################################################################

  ********************************** NOTICE **********************************
    You specified to use Taxonkit to add lineage info (set with the '-t' flag),
    but there were no taxids available (which typically come from inputs provided
    as NCBI accessions or GenBank files that might hold that information).

    In this case no lineage information could be added.
  *****************************************************************************

  ********************************** NOTICE **********************************
    You specified to swap labels either with a specific mapping file (passed
    to the '-m' flag) and/or by specifying to add NCBI or GTDB taxonomy info,
    but none of these worked out :(

    Genomes from your mapping file may have been filtered out, and there may
    have been no accessible taxonomy information to retrieve (which typically
    would come from inputs provided as NCBI accessions or GenBank files that
    might hold that information).

    In this case no labels could be swapped.
  *****************************************************************************
```

**Comment by AstrobioMike** [comment]
look into [this line](https://github.com/AstrobioMike/GToTree/blob/8053a391596a76adf9f3d908ad717046da4d70bb/bin/GToTree#L2672) when you can, where it's checking if there are taxids. The else of that if is to print the first "NOTICE" message above.

**Comment by AstrobioMike** [comment]
fixed as of v1.8.6
